src/components/index.js

```diff
@@ -9,6 +9,10 @@ import  {ProductProvider}  from "../components/context/product-context";
 import  CartNav  from "./CartComponents/cartNav/CartNav"
 import  CartCard  from "./CartComponents/CartCard/CartCard";
 import  BillCard  from "./CartComponents/CartCard/BillCard";
+import LoginCard from "./login_signup/LoginCard";
+import SignUpCard from "./login_signup/SignupCard";
+import { addToCart, deleteFromCart, updateQuantity } from "./services/cart-services/cart-functions";
+import { addToWishlist, removeFromWishlist } from "./services/wishlist-services/wishlist-functions";
```

Comment:
instead of importing and exporting you can directly use export in this file

Reply:
It's giving error for direct export, let me look into it